ddp-workspace/projects/ddp-sdk/src/lib/services/activity/activityConverter.service.ts

```diff
@@ -146,9 +146,7 @@ export class ActivityConverter {
         activityBlock.title = blockJson.title;
         activityBlock.headers = blockJson.headers;
         activityBlock.numberOfColumns = blockJson.numberOfColumns;
```

Comment:
Back-end implementation of a tabular block already uses that property as 'columnsCount'.
I guess we should rename it everywhere on UI, do you agree?

Reply:
Yeah. I made a comment on the fix to the API PR, but I don't think anybody saw it.
Regardless, columnsCount is not bad either. So not a problem as long as we sync up front and backend!